app/models/target.rb

```diff
@@ -11,6 +11,7 @@
 #  latitude   :decimal(, )      not null
 #  created_at :datetime         not null
 #  updated_at :datetime         not null
+#  matched    :boolean
```

Comment:
you are not using this attribute in your code.. maybe this PR still on WIP?

Reply:
Yes, this pr still on wip

Reply:
We should probably change this PR to draft then 🙂